src/app/shared/form/form.component.html

```diff
@@ -50,9 +50,9 @@
       <div class="form-group row">
 
         <div class="col text-right">
-          <button type="reset" class="btn btn-default" (click)="reset()">{{'form.cancel' | translate}}</button>
-          <button type="submit" class="btn btn-primary" (click)="onSubmit()"
-                  [disabled]="!(isValid() | async)">{{'form.submit' | translate}}
+          <button type="reset" class="btn btn-default" (click)="reset()">{{'Cancel' | translate}}</button>
+          <button type="submit" class="btn btn-primary" (click)="onSubmit($event)"
```

Comment:
Any reason why `$event` is included here now?

Reply:
no idea how that got in there. thanks, fixed today